src/libs/PolicyUtils.ts

```diff
@@ -378,6 +378,13 @@ function getPolicy(policyID: string | undefined): Policy | EmptyObject {
     return allPolicies[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`] ?? {};
 }
 
+function hasDependentTags(policy: OnyxEntry<Policy>, policyTagList: OnyxEntry<PolicyTagList>) {
+    return (
+        !!policy?.hasMultipleTagLists &&
+        Object.values(policyTagList ?? {}).some((tagList) => Object.values(tagList.tags).some((tag) => !!tag.rules?.parentTagsFilter || !!tag.parentTagsFilter))
+    );
```

Comment:
Fail fast for a bit more readability
```suggestion
    if(!policy?.hasMultipleTagLists) {
        return false;
    }
    return Object.values(policyTagList ?? {}).some((tagList) => {
        return Object.values(tagList.tags).some((tag) => !!tag.rules?.parentTagsFilter || !!tag.parentTagsFilter);
    });
```